tests/ert/unit_tests/ensemble_evaluator/test_ensemble_evaluator.py

```diff
@@ -407,11 +404,9 @@ async def mock_receiver(self):
         async with Monitor(url, token) as monitor:
             await asyncio.sleep(0.5)
             await monitor.signal_done()
-    # in 0.5 second we should receive at least 2 heartbeats
-    assert received_heartbeats > 1
+    assert received_heartbeats > 1, "we should have received at least 2 heartbeats!"
```

Comment:
Message is redundant as it says no more than what the fairly simple expression already says